### PR TITLE
Add news categories and fixes

### PR DESCRIPTION
**Pull request by andresriveratoro**

### CODE REVIEW

File: app/models/news_category.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@
 class NewsCategory < ApplicationRecord
   include Discard::Model
 
-  validates :call_to_action, :description, :media, :title, presence: true
+  validates :title, presence: true, uniqueness: true
```
Comment (andresriveratoro) [inline app/models/news_category.rb:6]: Added to prevent duplicate news categories

File: app/models/news_item.rb [hunk 2]
```diff
@@ -4,5 +4,6 @@ class NewsItem < ApplicationRecord
   include Discard::Model
 
   belongs_to :news_category
-  validates :date, :title, :url, presence: true
+  validates :date, :title, presence: true
+  validates :url, presence: true, uniqueness: true
```
Comment (andresriveratoro) [inline app/models/news_item.rb:8]: Added to prevent duplicate news items

File: db/migrate/20240327200012_change_media_type_in_news_categories.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class ChangeMediaTypeInNewsCategories < ActiveRecord::Migration[7.0]
+  def change
+    change_column :news_categories, :media, :boolean, using: 'media::boolean', default: false
```
Comment (andresriveratoro) [inline db/migrate/20240327200012_change_media_type_in_news_categories.rb:3]: media was incorrectly set to string

**Review by gileswells — APPROVED** [review]

**Review by gileswells — APPROVED** [review]
LGTM

**Pull request merged**